go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/terraform-providers/terraform-provider-tfe
 
 require (
 	github.com/hashicorp/go-hclog v0.0.0-20190109152822-4783caec6f2e // indirect
-	github.com/hashicorp/go-tfe v0.3.17
+	github.com/hashicorp/go-tfe v0.3.18-0.20190613152246-ce09432e1250
```

Comment:
We really should make sure we only use released versions in here.

Reply:
This is a WIP (hence the heading), so this will be all cleaned up after hashicorp/go-tfe#77 is merged.